src/kv.js

```diff
@@ -73,7 +73,8 @@ const trycid = string => {
   }
 }
 
-const create = async function * (map) {
+const create = async function * (map, targetSize = DEFAULT_TARGET_SIZE) {
+  if (targetSize < 2) throw ( new RangeError("targetSize must be greater than 1"))
```

Comment:
I'm not sure if this is the right way to error from here, but I also tried returning a Promise.reject and it also didn't do as expected...